test/visualTest.ts

```diff
@@ -275,8 +256,10 @@ describe("DualKpi", () => {
             };
 
             visualBuilder.updateFlushAllD3Transitions(dataView);
-            expect(visualBuilder.group.children("title").first().text().indexOf(topChartToolTipText) >= 0).toBeTruthy();
-            expect(visualBuilder.group.children("text.data-value").first().text().indexOf("k") < 0).toBeTruthy();
+            // TODO:// check if tooltip is shown
```

Comment:
what about this comment?

Reply:
Tooltip service shows tooltips above iframe, making them inaccessible. I am not able to query the tooltip elements in the DOM.